applications/asset_tracker/src/light_sensor/light_sensor.c

```diff
@@ -37,9 +39,16 @@ static struct ls_ch_data *ls_data[LS_CH__END] = { [LS_CH_RED] = &ls_ch_red,
 						  [LS_CH_IR] = &ls_ch_ir };
 static light_sensor_data_ready_cb ls_cb;
 static struct k_delayed_work ls_poller;
+static s32_t data_send_interval_s = CONFIG_LIGHT_SENSOR_DATA_SEND_INTERVAL;
+static bool initialized;
 
 static void light_sensor_poll_fn(struct k_work *work);
 
+static inline int submit_poll_work(const s32_t delay_s)
+{
+	return k_delayed_work_submit(&ls_poller, K_SECONDS(delay_s));
```

Comment:
Not exactly a change request, but I personally find code harder to read/understand when such simple operations are covered by wrapper functions.